### PR TITLE
feat(SPV-1258): add admin endpoint for confirm contacts

### PR DESCRIPTION
**Pull request by dzolt-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-go-client/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-go-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

**Review by mgosek-4chain — APPROVED** [review]

File: examples/README.md
Comment (chris-4chain) [inline examples/README.md:1]: I think that different IDE has different auto-formaters for `md` files. That's why so many changes in this file. 
- Maybe we could establish some one format and configure our IDEs to be consistent.

Just saying 🤷
To me, it's not a blocker 😄

**Review by chris-4chain — APPROVED** [review]

File: examples/README.md
Comment (dzolt-4chain) [reply examples/README.md:1]: Yeah I totally agree. But please let's do it in different pr 😅

Comment (chris-4chain) [reply examples/README.md:1]: Of course - that's why I approved this PR 👍

**Review by jakubmkowalski — APPROVED** [review]

**Comment by jakubmkowalski** [comment]
Tested

**Pull request merged**